xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -527,17 +542,17 @@ $xwiki.ssx.use('XWiki.TagCloud')##
 ##
 ## Display tag cloud.
 {{html}}
-#if ($tagCount.size() &gt; 0)
+#if ($tagCountListSorted.size() &gt; 0)
   &lt;ol class="tagCloud"&gt;
-  #foreach ($tagEntry in $tagCount.entrySet())
+  #foreach ($tagEntry in $collectiontool.sort($tagCountListSorted, 'name'))
```

Comment:
Isn't this changing the way tags are displayed? Isn't that a behavior change compared to before? If so, this issue is not just about limiting but also about the order tags are displayed and we'll need to document this too.

Reply:
No, precisely I did this to match the way tag were already displayed! Tags are listed alphabetically, independently of their occurrence. But `$tagCountListSorted` is instead sorted by occurrence first, so that's why I'm reordering the items here

Reply:
ok great, thanks!